src/ucp/wireup/select.c

```diff
@@ -389,6 +390,10 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
             continue;
         }
 
+        if (select_params->ep_init_flags & UCP_EP_INIT_CONNECT_TO_IFACE_ONLY) {
+            local_iface_flags |= UCT_IFACE_FLAG_CONNECT_TO_IFACE;
```

Comment:
where is it used?

Reply:
for EP created for sending WIREUP/EP_REMOVED packet as a reply on KA

Reply:
i meant the var `local_iface_flags`

Reply:
fixed